text/0040-pagination-convention.md

```diff
@@ -0,0 +1,47 @@
+- Title: Pagination Convention
+- Start Date: 2021-05-10
+- Specification PR: [#40](https://github.com/meilisearch/specifications/pull/40)
+- MeiliSearch Tracking-Issues:
+
+# Pagination Convention
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+Pagination allows the content to be viewed in the form of a sequence in order to facilitate the browsing of its content by not overloading the number of elements to be viewed for the user. Pagination also has the advantage of easily providing the total number of items that remain to be traversed by not overloading the network with all the content that can be loaded.
+
+### II. Motivation
+
+This specification will be used as a convention to implement simple and standard paginations to be set up on API routes such as the list of updates for example. Since the pagination of search results is intrinsically linked to technical implementations of the search engine, this specific case is detailed in its own specification.
+
+### III. Additional Materials
+N/A
+
+### IV. Explanation
+
+##### Requirements
```

Comment:
How to trigger the pagination? I guess it's explained in the previous spec, but nothing mentions it here if I'm correct.
Why do we need 2 specs? 🙂 I'm sure there is a reason, but I feel a little bit confused at the moment.

Reply:
Pagination is enabled through the use of offset and limit. By default, we can say that the paging is always triggered since we return a limited number of results from the first call.